src/content/docs/en/recipes/docker.mdx

```diff
@@ -31,7 +31,7 @@ WORKDIR /app
 COPY . .
 
 RUN npm install
-RUN npm run build
+RUN npm build
```

Comment:
Pretty sure this has to stay `npm run build`. `npm build` is an internal command an not what you specified in `package.json` scripts.

```suggestion
RUN npm run build
```